lib/cocoapods/installer/xcode/pods_project_generator/app_host_installer.rb

```diff
@@ -62,7 +62,8 @@ class AppHostInstaller
           # @param [Hash] info_plist_entries see #info_plist_entries
           # @param [String] product_basename see #product_basename
           #
-          def initialize(sandbox, project, platform, subgroup_name, group_name, app_target_label, add_main: true, add_launchscreen_storyboard: platform == :ios, info_plist_entries: {}, product_basename: nil)
+          def initialize(sandbox, project, platform, subgroup_name, group_name, app_target_label, add_main: true,
```

Comment:
just formatting because I was inspecting these sources and its annoying to scroll. Irrelevant to the fix ultimately.